src/scss/components/_header.scss

```diff
@@ -0,0 +1,95 @@
+.header {
+  position: sticky;
+  top: 0;
+  background: var(--white);
+
```

Comment:
Remember to handle the `z-index` levels using variables so that the navigation is shown over other elements, and not behind.